src/align.py

```diff
@@ -16,7 +16,46 @@ def align(x: str, y: str, edits: str) -> tuple[str, str]:
     ('ACCACAGT-CATA', 'A-CAGAGTACAAA')
 
     """
-    return "", ""
+
+    # Better variable name, I think
+    seq1 = x
+    seq2 = y
+
+    # The two aligned files are saved at there variables
+    align1 = ""
+    align2 = ""
+
+    # Extra int adds to i, if a gap has accured. Otherwise the alignments are not recreated properly
+    extra_int1 = 0 
+    extra_int2 = 0
+
+    for i in range(len(edits)):
```

Comment:
I think this would be simpler if you didn't iterate over indices here, but just the operations:

```python
for op in edit:
    ...
```

Then, everwhere you check `edites[i]` you would use `op`.

Of course, the `extra_int`*N* variables would have to move from 0 to the length of the aligned strings instead, but you would just use `i, j = 0, 0` that you increment every time you append to `align1` or `align2`, respectively.